modules/jaeger-config-query.adoc

```diff
@@ -34,11 +26,32 @@ Query is a service that retrieves traces from storage and hosts the user interfa
 |/{path}
 |
 
+|query:
+  replicas:
+|Specifies the number of Query replicas to create.
+|Integer, for example, `2`
+|
+
 |query:
   port:
 |The port for the query service.
 |
 |16686
```

Comment:
You are right again, @jkandasa. We have two options here:

- document here that users will need to manage the services by themselves in case they decide to use a custom port
- not mention this option, as it's not something users would normally do. I can't think of a use case for it, to be honest...

Reply:
@jpkrohling  So delete the "port" row sounds like probably the best option?

Reply:
Deleting it sounds good to me.